src/IntegratedExperts/BehatScreenshot/ScreenshotContext.php

```diff
@@ -74,6 +82,29 @@ public function __construct($parameters = [])
     }
 
     /**
+     * Init function before tests run.
+     *
+     * @param BeforeSuiteScope $scope
+     *
+     * @BeforeSuite
+     */
+    public static function init(BeforeSuiteScope $scope)
+    {
+        foreach ($scope->getSuite()->getSetting('contexts') as $context) {
```

Comment:
I'm still not satisfied with this logic. It should be quite simple. Also, `$contextSettings` is not going to be defined if the setting was not found.